MSAL/test/unit/ios/external-cache/MSALLegacySharedAccountsProviderTests.m

```diff
@@ -464,7 +490,45 @@ - (void)testRemoveAccount_whenTenantProfilesPassed_andMatchingAccountsPresent_sh
     XCTAssertEqualObjects(v3Blob[accountId][@"username"], @"user@contoso.com");
 }
 
-- (void)testRemoveAccount_whenNonMatchingAccountPresent_shouldOnlyUpdateTimestamps
+- (void)testRemoveAccount_whenWipeAccountYES_whenTenantProfilesPassed_andMatchingAccountsPresent_shouldRemoveEntireMatchingAccountBlob
```

Comment:
Looks like this test case achieves the same end result as testRemoveAccount_whenWipeAccountYES_whenMatchingAccountsPresent_shouldRemoveEntireAccountBlob, with the only difference being the tenantIdentifier in above scenario is accountClaims[@"oid"]. 
Could you please provide the E2E scenario where the tenant identifier is not available, so the user has to rely on accountClaims?

Reply:
One test case has wipeAccount == NO and this one has wipeAccount == YES. 
When user is asking to be "forgotten", we need to remove all artifacts we have written. 

I'm not sure about accountClaims. Let's follow up offline on that.